examples/checkpoint_FNO_darcy.py

```diff
@@ -95,20 +88,18 @@
               optimizer=optimizer,
               scheduler=scheduler, 
               regularizer=False, 
-              training_loss=train_loss)
+              training_loss=train_loss, 
+              save_every=1,
```

Comment:
Perhaps we don't save by default here? This may create a lot of disk for unsuspecting users. 

I'd be for a clear documentation and no default saving.

Reply:
That sounds good.

Reply:
We don't save by default. You have to pass a value to `save_every` for the checkpointing (default is `None`)